tornado_server.py

```diff
@@ -75,6 +78,20 @@ def check_origin(self,origin):
 ])
 
 if __name__ == "__main__":
-    http_server = tornado.httpserver.HTTPServer(application)
+ 
+    # This line will connect to the website, read its contents
+    # and parse the JSON output
+	
+    data = loads(urlopen("http://httpbin.org/ip").read())
+    public_ip = data["origin"]
+    print "The public IP is : %s" % data["origin"]
+    if str(public_ip) == "52.11.133.36":
```

Comment:
Create secure https Python server if we are hosting on aws. Otherwise create regular http Python server.